en/releases/1.14.md

```diff
@@ -1,4 +1,10 @@
-## PX4-Autopilot v1.14 Release Notes
+# PX4-Autopilot v1.14 Release Notes
+
+## Upgrading Notes
+
+Below important notes when upgrading to v1.14 are listed:
+
+1. Default disarmed PWM was changed from 900us to 1000us. So make sure to check whether you were using default PWM disarmed values, and if it works with your setup. You can find related information in the [step 7 of ESC calibration](../advanced_config/esc_calibration.md#steps) document for details.
```

Comment:
@MaEtUgR is this a good way to phrase it?

Reply:
There are two parts to this change: the ESC calibration defaults now set the range from (1000us - 2000us), and the default PWM settings change from disarmed (900 to 1000), min (1000 to 1100), max (2000 to 1900). The change means that ESC calibration is consistent and any calibrated ESC should now work out of the box with the default ranges.

BUT Vehicle parameters survive a firmware update, unless (possibly) they are renamed. So AFAIK there should be no impact to an existing user of this particular change, because there are no new parameters. Unless you're saying that it will use the new defaults unless the values were changed by the user previously?

If the default values are not changed then we should change the tone of this to "Highly recommending" that users rerun ESC calibration and use the new defaults. @MaEtUgR What about this?